src/liballoc/raw_vec.rs

```diff
@@ -235,7 +235,66 @@ impl<T, A: Alloc> RawVec<T, A> {
         }
     }
 
-    /// Doubles the size of the type's backing allocation. This is common enough
+    /// Grows the vector capacity by `capacity_increase`.
```

Comment:
Doc should probably say it's growing for at least this capacity_increase, adjusted by the growth policy. 

A postcondition is that if the function returns, at least `capacity + capacity_increase` memory is allocated and that sum does not wrap around.